packages/renderer/src/get-local-chromium-executable.ts

```diff
@@ -0,0 +1,25 @@
+import puppeteer from 'puppeteer-core';
+import puppeteer_node from 'puppeteer-core/lib/cjs/puppeteer/node';
+import {downloadBrowser} from 'puppeteer-core/lib/cjs/puppeteer/node/install';
+
+const getLocalRevision = (): puppeteer.BrowserFetcherRevisionInfo => {
+	const productName = 'chrome';
+	const browserFetcher = puppeteer_node.createBrowserFetcher({
+		product: productName,
+		host: 'https://storage.googleapis.com',
+	});
+	const revisionInfo = browserFetcher.revisionInfo('848005');
```

Comment:
Pretty impressive!

So this will only work if someone has exactly Chrome 84.8005 installed though? 🤔  Might be a bit too strict, maybe better to check for path (e.g. /Applications/Google Chrome.app)

Reply:
actually 848005 corresponds to Chromium 90.0.4403.0.

To check for executable path for chrome or chromium for each os is a bit complicated than it seems. Figuring out how to write the script for chrome or chromium executable for each os.

Currently how it works is that if you don't have the chormium executable in `puppeteer-core` it downloads the default `Chromium 90.0.4403.0` in `node-modules/puppeteer-core/.local-chromium` and from the next time it automatically detects the executable path in `.local-chromium` and use that chromium executable.